index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+          <a href="" id="logo">Ikonde Arnold Nekemiah</a>
+        </div>
+        <nav id="navi" class="dropdown">
+            <img class="menu" id="btn-menu" src="./Images/Icon - Menu.png" alt="menu">
+        </nav>
```

Comment:
- [ ] This is a width of 376px and I could not see your hamburger, I advise you to use flexbox to style it.
![image](https://user-images.githubusercontent.com/86200356/234809128-a549f44a-9455-4b55-8f48-96eeb40943d6.png)